packages/experience/src/apis/experience/index.ts

```diff
@@ -0,0 +1,149 @@
+import {
+  InteractionEvent,
+  type PasswordVerificationPayload,
+  SignInIdentifier,
+  type VerificationCodeIdentifier,
+} from '@logto/schemas';
+
+import { type ContinueFlowInteractionEvent } from '@/types';
+
+import api from '../api';
+
+import { experienceApiRoutes, type VerificationResponse } from './const';
+import {
+  initInteraction,
+  identifyUser,
+  submitInteraction,
+  updateInteractionEvent,
+  _updateProfile,
+  identifyAndSubmitInteraction,
+} from './interaction';
+
+export {
+  initInteraction,
+  submitInteraction,
+  identifyUser,
+  identifyAndSubmitInteraction,
+} from './interaction';
+
+export * from './mfa';
+export * from './social';
+
+export const registerWithVerifiedIdentifier = async (verificationId: string) => {
+  await updateInteractionEvent(InteractionEvent.Register);
```

Comment:
can we make an experience api SDK and use it for all API calls later?

Reply:
Yes,  create an issue tracking this. Consider gradually migrating and maybe with new elements based components.

Reply:
can you paste the issue number here?

Reply:
LOG-9941